matchmaker/assign.py

```diff
@@ -4,6 +4,17 @@
 Assigners are expected to implement different assignment strategies, potentially
 creating different team compositions.
 '''
+from copy import deepcopy
+
+
+def evaluate_SSE(teams):
```

Comment:
Isn't clear what SSE means here.

Reply:
Agreed, worth including a docstring. (Sum of squared errors)